spec/Auth.spec.js

```diff
@@ -81,17 +81,6 @@ describe('Auth', () => {
         .then(roles => expect(roles).toEqual([]))
         .then(() => done());
     });
-
-    it('should properly handle bcrypt upgrade', done => {
```

Comment:
This is an old test added 8+ years ago that was introduced in #2549. I don't believe it's needed anymore.